modules/userId/index.js

```diff
@@ -660,6 +703,8 @@ export function init(config) {
   // exposing getUserIds function in global-name-space so that userIds stored in Prebid can be used by external codes.
   (getGlobal()).getUserIds = getUserIds;
   (getGlobal()).getUserIdsAsEids = getUserIdsAsEids;
+
+  (getGlobal()).refreshUserIds = refreshUserIds;
```

Comment:
no need for an extra line here IMO

```suggestion
  (getGlobal()).refreshUserIds = refreshUserIds;
```